extensions/gdx-tools/src/com/badlogic/gdx/tools/texturepacker/TexturePacker.java

```diff
@@ -719,7 +717,7 @@ static public String getAtlasName (String name, boolean flattenPaths) {
 		}
 	}
 
-	static public enum Resampling {
+	public enum Resampling {

```

Comment:
I know IDEA gives some redundancy notice about how inner `enum`s are always `static`, and don't need the static modifier, but it can be a little tricky to actually know that classes aren't always static, but enums are. I'm fine with this how it is here, but I'd also be fine if the `static` stayed.